src/Microsoft.Graph/Generated/requests/GraphServiceClient.cs

```diff
@@ -72,17 +72,6 @@ public partial class GraphServiceClient : BaseClient
         {
         }
     
-        /// <summary>
-        /// Gets the GraphServiceWorkbooks request builder.
-        /// </summary>
-        public virtual IGraphServiceWorkbooksCollectionRequestBuilder Workbooks
```

Comment:
This is a fix for the ghost path via https://github.com/microsoftgraph/msgraph-metadata/pull/247